apps/mobile/src/components/FilterOffersScreen/atom.ts

```diff
@@ -114,6 +121,46 @@ export const amountTopLimitAtom = focusAtom(offersFilterAtom, (optic) =>
   optic.prop('amountTopLimit')
 )
 
+export const locationSuggestionsAtom = atom<LocationSuggestion[]>([])
+
+export const locationSuggestionsAtomsAtom = splitAtom(locationSuggestionsAtom)
+
+export const getLocationSuggestionsActionAtom = atom(
```

Comment:
The name is missleading, this refreshes locationSuggestionsAtom. How about renaming it to something like `updateAndRefreshLocationSuggestionsActionAtom` or `triggerLocationSuggestionsQueryActionAtom`)